iphone/cli/commands/_build.js

```diff
@@ -4368,8 +4378,11 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 			writeAssetContentsFile.call(this, path.join(appIconSetDir, 'Contents.json'), appIconSet);
 
 			if (missingIcons.length) {
-				if (defaultIconHasAlpha) {
-					return next(new Error(__('DefaultIcon.png cannot be used because it contains an alpha channel')));
+				if (defaultIcon && defaultIconHasAlpha) {
+					this.logger.error(__('%s cannot be used because it contains an alpha channel', defaultIcon));
+					this.logger.error(__('Create a image named "%s" in the root of your project', 'DefaultIcon-ios.png'));
```

Comment:
[ERROR] Create a image named "DefaultIcon-ios.png" that does not contain an alpha channel in the root of your project